### PR TITLE
Allows to skip E2E tests based on bash script condition.

### PR DESCRIPTION
**Pull request by artursouza**
E2E tests on ARM64 are still flaky. This allows work to stabilize it to happen via PRs and only enable it again once proved stable for the first time.

Please reference the issue this PR will close: N/A
## Checklist

Please make sure you've  completed the relevant tasks for this PR, out of the following list:

* [ ] Code compiles correctly
* [ ] Created/updated tests
* [ ] Unit tests passing
* [ ] End-to-end tests passing
* [ ] Extended the documentation / Created issue in the https://github.com/dapr/docs/ repo: dapr/docs#_[issue number]_
* [ ] Specification has been updated / Created issue in the https://github.com/dapr/docs/ repo: dapr/docs#_[issue number]_
* [ ] Provided sample for the feature / Created issue in the https://github.com/dapr/docs/ repo: dapr/docs#_[issue number]_

### CODE REVIEW

File: .github/workflows/dapr-test.yml
Comment (halspang) [inline .github/workflows/dapr-test.yml:541]: Why not remove it here: https://github.com/dapr/dapr/blob/master/.github/workflows/dapr-test.yml#L409

Or just use the already defined variables instead of making a script? Seems like this way will spend a lot of time doing all the setup work only to not run tests.

**Review by halspang — CHANGES_REQUESTED** [review]
This seems like a more complicated way of doing this. What's the backing behind it?

**Review by yaron2 — APPROVED** [review]

File: .github/workflows/dapr-test.yml
Comment (artursouza) [reply .github/workflows/dapr-test.yml:541]: I have explained why as comments in the script. It will allow us to enable in a PR and run /ok-to-test multiple times to validate the stability before merging to master. Changes to the workflow YAML do not affect /ok-to-test unless merged into master.

Comment (halspang) [reply .github/workflows/dapr-test.yml:541]: > # This script allows the stabilization to take place prior to merging into master.

This wasn't super clear to me that it was unblocking /ok-to-test. Mentioning it there would be good, but I won't block on it since we have the record in the PR.

**Review by halspang — APPROVED** [review]

**Pull request merged**
